src/components/JumbotronVideoComponent.vue

```diff
@@ -9,7 +9,13 @@ const props = defineProps<{
   /**
    * If set, the main element will have a background video.
    */
-  video?: string;
+  videos?: string[];
+  /**
+   * Set the orientation of the spliting.
+   * For the split module
+   * @defaultValue `'vertical'`
```

Comment:
\@ defaultvalue is not valid tsdoc

Reply:
I used this as reference https://tsdoc.org/pages/tags/defaultvalue/
but you mean this https://jsdoc.app/tags-param.html#optional-parameters-and-default-values.
Good